build.gradle

```diff
@@ -1,16 +1,24 @@
 plugins {
     id 'java'
 }
+
+group 'org.example'
+version '1.0-SNAPSHOT'
+
 repositories {
     mavenCentral()
 }
+
 dependencies {
-    testImplementation 'com.codeborne:selenide:7.5.0'
-    testImplementation group: 'org.testng', name: 'testng', version: '7.10.2'
-    testImplementation 'junit:junit:4.13.2'
-    testImplementation 'org.slf4j:slf4j-simple:2.0.7'
+    implementation 'com.codeborne:selenide:6.19.1' // Core Selenide library
```

Comment:
Seems like you're fan of changing this file :D
In merged Pull Request you had enough implementations, here again you have either more or less..

`    implementation 'com.codeborne:selenide:6.18.0'
    testImplementation 'org.testng:testng:7.9.0'
    implementation 'org.slf4j:slf4j-simple:2.0.9'
`